ddtrace/_trace/span.py

```diff
@@ -595,6 +595,22 @@ def context(self) -> Context:
             self._context = Context(trace_id=self.trace_id, span_id=self.span_id, is_remote=False)
         return self._context
 
+    @property
+    # This can't actually return None, but we say it can to satisfy type checking.
+    def _local_root(self) -> Optional["Span"]:
```

Comment:
I would consider making this `-> "Span"`, we'll need to then check the places in code that do `span._local_root is None` to see if they actually are expecting a `None` value, or if they are just checking it because the original definition of `_local_root` was `Optional["Span"]`

Reply:
Done 👍